File: string-handling/string-concatenation/python/message_alert.py
# Raspberry Pi Foundation
# Developed to be used alongside Isaac Computer Science,
# part of the National Centre for Computing Education
# Usage licensed under CC BY-SA 4

def message_alert(new_messages):
    """Example of type conversion for string concatenation"""    
    print("You have " + str(new_messages) + " new messages")
    

# This code will run if this file is executed directly
# (i.e. not called by another program)
if __name__ == "__main__":
    messages = 5
    message_alert(messages)

File: string-handling/string-concatenation/c-sharp/message_alert.cs
/*
Raspberry Pi Foundation
Developed to be used alongside Isaac Computer Science, part of the National Centre for Computing Education
Usage licensed under CC BY-SA 4

Note: This file is designed to be copied out and compiled on your machine.
In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file.
To run this file you need to:
1. Copy the contents
2. Paste them into the C# IDE of your choice (Visual Studio, for example)
3. Change the namespace to match your project (if necessary)
4. Compile the program
5. Run the program
*/

using System;

namespace IsaacCodeSamples
{
    
    class StringConcatenation
    {
        
        // The main method is the entry point for all C# programs
        public static void Main() {
            int newMessages = 5;
            MessageAlert(newMessages);           
        }

        
        //Example of type conversion for string concatenation
        public static void MessageAlert(int newMessages) {            
            Console.WriteLine("You have " + newMessages.ToString() + " new messages");
        }
        
        
    }
}
